app/src/components/Accordion/theme.ts

```diff
@@ -15,7 +15,7 @@ export const accordionTheme: keepAccordionTheme = {
   icon: {
     base: 'transition-transform duration-300',
     rotated: {
-      full: 'rotate-180',
+      full: 'rotate-45',
```

Comment:
Please change it with 'rotate-[225deg]' instead of 'rotate-45' for a similar effect with a slightly different view.

Reply:
Resolved check now